src/components/search-bar/SearchBar.jsx

```diff
@@ -130,19 +130,21 @@ const AdvancedSection = observer(({
       <Text c="elv-gray.8" size="xl" fw={700} mb={8}>Version</Text>
       <Radio.Group
         value={searchStore.searchHostname}
-        defaultValue="ai"
+        defaultValue="ai.contentfabric.io"
         onChange={(value) => {
           searchStore.SetSearchHostname({host: value});
+          summaryStore.SetMlcacheHostname({host: value});
+          highlightsStore.SetMlcacheHostname({host: value});
```

Comment:
So summary, highlights, tags, and search still have potential to use different hostnames?